tests/examples/contracts/intelligent_oracle.py

```
@@ -139,7 +140,7 @@ def resolve(self, evidence_url: str = "") -> None:
         for resource_url in resources_to_check:
 
             def evaluate_single_source() -> str:
-                resource_web_data = gl.get_webpage(resource_url, mode="text")
+                resource_web_data = gl.nondet.web.render(resource_url, mode="text")
```

Ruff (0.12.2)

`gl` may be undefined, or defined from star imports

(F405)